File: Assignment51to53/program6.java
//write java program which accept number of rows and number of columns from user and display below pattern.

import java.lang.*;
import java.util.*;

class pattern
{
    public void pattern(int iRow,int iCol)
    {   
        int i = 0,j = 0;
        char ch= '\0',ch2 = '\0';


        for(i = 1,ch = 'A'; i<= iRow; i++,ch++)
        {
            for(j = 1,ch2 = 'A'; j <= iCol; j++,ch2++)
            {
                System.out.print(ch2+"\t");
            }
            System.out.println();
        }
    }
}
class program6
{
    public static void main(String arg[])
    {
        pattern obj = new pattern();
        Scanner sobj = new Scanner(System.in);

      System.out.println("Enter the Row Number..");
        int iValue = sobj.nextInt();

        System.out.println("Enter the Columns Number...");
        int iValue2 = sobj.nextInt();

        obj.pattern(iValue,iValue2);
    }
}

File: Assignment3638/program6.c
//write a recursive program which accept number from user and display below pattern
// 5 * 4 * 3 * 2 * 1 *
#include<stdio.h>

void Display(int iNo)
{
    static int i = 0;
    static int k = 1;
    if(i < iNo)
    {
     printf("%d   * \t",k);
        k++;
        iNo--;
        Display(iNo);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter the Number\n");
    scanf("%d",&iValue);

    Display(iValue);
}